libbpf-rs/src/perf_buffer.rs

```diff
@@ -142,6 +161,49 @@ impl<'a, 'b> PerfBufferBuilder<'a, 'b> {
         }
     }
 
+    unsafe extern "C" fn call_event_cb(
+        ctx: *mut c_void,
+        cpu: i32,
+        hdr: *mut libbpf_sys::perf_event_header,
+    ) -> i32 {
+        #[repr(C)]
+        #[derive(Debug, Default, Copy, Clone)]
```

Comment:
Shall we minimize the number of derives to what is actually necessary?

Reply:
Done